tests/onion/test_multi.py

```diff
@@ -5,11 +5,14 @@
 from pathlib import Path
 from typing import Generator
 
+import matplotlib as mpl
 import numpy as np
 import pytest
 
 import dynsight
 
+mpl.use("Agg")
```

Comment:
@matteobecchi why do we need plots in tests, are we checking them?

Reply:
The problem with this is that Onion code doesn't belong to Dynsight. It calls the `main.py` and `main_2d.py` where `mpl` is imported.

Reply:
Plots are not checked in tests, but when onion (in the "main" version) is run, it does them anyway, you can't avoid it.